### DISCUSSION TITLE
Reven python API on a client workstation installation issue

### DISCUSSION THREAD

**Discussion by ManuSecsearch** [tetrane/reven-community#20] [Q&A] [answered]
Hi guys 
Hope you're all doing good.
I'm currently trying to install the Reven Api in win10 and i'm facing the following error when trying to execute the 
'python.exe -m pip install (Get-Item reven_api*.whl).FullName'

> ERROR: reven_api-2.10.2-cp37-cp37m-win_amd64.whl is not a supported wheel on this platform.

My env is 
(env) PS C:\Users\vrtus\Documents\work\reversing\reven\REVEN-2.10.2-python3.7-windows> python
Python 3.9.7 (tags/v3.9.7:1016ef3, Aug 30 2021, 20:19:38) [MSC v.1929 64 bit (AMD64)] on win32
Type "help", "copyright", "credits" or "license" for more information.
>>>

Am i missing something ?

**Comment by Hyask** [comment, marked as answer]
Hi there,

Indeed, the REVEN Python API on Windows only supports Python 3.7. That's something we want to improve in the future,
especially since 3.7 starts to age a bit, but that's unfortunately the only Python we build against for now.

If that's a problem for you, don't hesitate to tell us, so we can review the priority on this matter.

Hope this helps

**Reply by ManuSecsearch** [reply]
Oh yes i missed that detail  :)  thanks
